package.json

```diff
@@ -29,7 +29,9 @@
     "collectCoverage": true,
     "coverageDirectory": "coverage",
     "testEnvironment": "jsdom",
-    "setupFiles": ["./someModuleForTest"],
+    "setupFiles": [
+      "./someModuleForTest"
+    ],
```

Comment:
are the unit test cases done for these changes ?

Reply:
yes, written